extra/1-radio-stations.js

```diff
@@ -25,7 +28,15 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations(){
+  let realStations = []
+  for (i in getAllFrequencies()) {
+    if (isRadioStation(getAllFrequencies()[i]) == true){
```

Comment:
the true in the if statement can also be omitted and be written as:
`if (isRadioStation(getAllFrequencies()[i]))`